config/default.yml

```diff
@@ -468,6 +468,8 @@ Layout/EmptyLineBetweenDefs:
   AllowAdjacentOneLineDefs: false
   # Can be array to specify minimum and maximum number of empty lines, e.g. [1, 2]
   NumberOfEmptyLines: 1
+  ApplyToClassDefs: false
```

Comment:
Perhaps we can name this `EmptyLineBetweenClassDefs` instead? Probably it'd be best if there was also a configuration options for methods (`EmptyLineBetweenMethodDefs`) so this cop would get more flexible. 

I'd also enable those new options by default, as I doubt they'll cause a lot of offenses. The version changed should be updated to `<<next>>` as well.